server/knex/seeds/eote_force_powers.ts

```diff
@@ -372,159 +399,163 @@ exports.seed = function(knex)
+                //     description: '',
+                //     min_rating: 1,
+                //     upgrades: JSON.stringify({
+                //         strength: {
+                //             available: 0,
+                //             description: ''
+                //         },
+                //         magnitude: {
+                //             available: 0,
+                //             description: ''
+                //         },
+                //         duration: {
+                //             available: 0,
+                //             description: ''
+                //         },
+                //         range: {
+                //             available: 0,
+                //             description: ''
+                //         },
+                //         mastery: {
+                //             available: 0,
+                //             description: ''
+                //         },
+                //         control: [
+                //             {
+                //                 description: ''
+                //             },
+                //             {
+                //                 description: ''
+                //             }
+                //         ]
+                //     }),
+                //     reference: 'F-CRB:302',
+                //     official: true,
+                //     scope: 'public'
+                // },
+                // {
+                //     id: 11,
+                //     name: 'Sense',
+                //     description: '',
+                //     min_rating: 1,
+                //     upgrades: JSON.stringify({
+                //         strength: {
+                //             available: 0,
+                //             description: ''
+                //         },
+                //         magnitude: {
+                //             available: 0,
+                //             description: ''
+                //         },
+                //         duration: {
+                //             available: 0,
+                //             description: ''
+                //         },
+                //         range: {
+                //             available: 0,
+                //             description: ''
+                //         },
+                //         mastery: {
+                //             available: 0,
+                //             description: ''
+                //         },
+                //         control: [
+                //             {
+                //                 description: ''
+                //             },
+                //             {
+                //                 description: ''
+                //             }
+                //         ]
+                //     }),
+                //     reference: 'F-CRB:304',
+                //     official: true,
+                //     scope: 'public'
+                // }
```

Comment:
Commented out code?

Reply:
About to commit that filled in, and then this PR's totally done.